pynicotine/events.py

```diff
@@ -182,6 +182,8 @@
     "upload-denied",
     "upload-failed",
     "upload-file-error",
+    "uploads-start-shutdown",
+    "uploads-stop-shutdown",
```

Comment:
This contradiction in terms is rather odd. Suggest:

`"uploads-shutdown-requested"`
`"uploads-shutdown-cancelled"`

or simply (the event doesn't necessarily have to be exclusive to the uploads module):

`"shutdown-request"` (...`ed`)
`"shutdown-cancel"` (...`led`)

Reply:
`uploads-shutdown-request` and `uploads-shutdown-cancel` perhaps? I want to make it clear that the request came from the uploads module, so we can react accordingly in the GUI. We don't want to cause confusion between this event and the existing `schedule-quit`.

Reply:
Yes, go for that.